test/e2e/storage/utils/utils.go

```diff
@@ -656,33 +656,55 @@ func genBinDataFromSeed(len int, seed int64) []byte {
 }
 
 // CheckReadFromPath validate that file can be properly read.
-func CheckReadFromPath(f *framework.Framework, pod *v1.Pod, volMode v1.PersistentVolumeMode, path string, len int, seed int64) {
+//
+// Note: disableCache only affects BlockMode PVCs and does not work with
+// (default) BusyBox Pods.
+func CheckReadFromPath(f *framework.Framework, pod *v1.Pod, volMode v1.PersistentVolumeMode, disableCache bool, path string, len int, seed int64) {
 	var pathForVolMode string
+	nocache := ""
 	if volMode == v1.PersistentVolumeBlock {
 		pathForVolMode = path
+		if disableCache {
+			// TODO: "len" (dd: bs+count) need to be adjusted to
+			// sector(s). Make sure CheckWriteToPath() wrote enough
+			// bytes!
+			len = 512
+			nocache = "iflag=direct"
```

Comment:
Should we consider renaming to `iflag`?  
(Then the one in `CheckWriteToPath` should be `oflag`.)

Reply:
I have adapted the names of the variables, `nocache bool` and `directIO bool` as arguments, `oflag` and `iflag` as parameters to `dd`.